requirements.txt

```diff
@@ -1,5 +1,6 @@
-fastapi==0.67.0
```

Comment:
@Brvilardi @Renan-Reschke 

Eu retirei as versoes das nossas dependencias, pra sempre termos o mais atualizado, vcs vem algum problema?

Reply:
1) Se tiver um update que nao conhecemos talvez quebre o codigo ou dê algum comportamento esquisito, mas acho que a chance é baixa.

2) Um ponto mais preocupante é de ter mss rodando em prod com versoes de dependencia diferente:
- Os containers no deploy vao ter a versao disponível no momento do build da imagem 

Tipo, pensa que a gente atualizou o mss_gerenciamento_usuarios hoje e a versao do fastApi era 0.67.0. - os containers de mss_gerenciamento_usuarios vao ter a versao 0.67.0 até que uma nova imagem seja buildada e deployada

Daqui uma semana a gente atualiza o mss_autenticacao e o fastapi ta em 0.68.0

Daqui duas semanas a gente atualiza o mss_materias e o fastapi ta em 0.69.0

Ou seja, daqui 2 semanas cada mss tá com uma versao diferente de fastapi, talvez isso possa gerar uns comportamentos esquisitos, principalmente se o numero de dependencias compartilhadas entre os mss for grande.


Mas não sei se isso é realmente um problema ou se é um pouco de frescura da minha cabeça

Reply:
Eu acho melhor deixar uma versão fixa e quando houver atualização de dependência podemos optar por atualizar tudo de uma vez (ou não), justamente pra evitar o problema que o Bruno comentou.

Reply:
Justíssimo, melhor deixar todos os mss na mesma versão e a gnt atualizar na mão qdo for necessário. Vou voltar com as versões.